README.md

```diff
@@ -1,108 +1,107 @@
 # Bridge TensorFlow* to run on Intel® nGraph™ backends
 
 This directory contains the code needed to build a TensorFlow 
-plugin that can be used with Intel® nGraph™. nGraph is an [open-source C++ library, compiler and runtime] that provides developers with the means to 
-train and run DNN models efficiently on custom backends: GPU, CPU, and custom 
-silicon.
+plugin to the Intel® nGraph™ Compiler. As part of a TensorFlow 
+toolchain, nGraph can speed up training workloads using CPU; or 
+the nGraph Library and runtime suite can be used to customize 
+and deploy Deep Learning inference models that will "just work" 
+with variety of nGraph-enabled backends: CPU, GPU, and custom 
+silicon.  
 
-## Linux instructions
+*   [Linux instructions](#linux-instructions)
+*   [OS X instructions](#os-x-instructions)
+*   [Debugging](#debugging)
+*   [Support](#support)
+*   [About nGraph](#about-ngraph)
 
```

Comment:
This link is not working in the README. Maybe because the section is called 'About Intel® nGraph™' and the link says '#about-ngraph'?